src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -424,7 +424,7 @@ export default {
 				// Add handlers so the Actions menu opens on hover
 				nativeOn: {
 					dragstart: this.dragStart,
-					dragenter: () => { this.$refs.actions.opened = true },
+					dragenter: () => { this.$refs.actions.open = true },
```

Comment:
I don't think you can change the prop from the outside, no?

Reply:
There is no prop anymore, just a data variable. Is this not what you meant? If not, I will keep my fingers off the code and let you do the last changes 🙈

Reply:
Nono, I meant that open is a prop passed to the BreadCrumb component, right?
You cannot mutate prop directly, prop are not a data variable, they're supposed to just be a reference to an existing variable. 

I'm expecting Vue to throw "Vue Error: Avoid Mutating a Prop Directly"

Reply:
Yes, if `open` was a prop, there would be the error you mentioned. But open is in the data function:

```
	data() {
		return {
			/**
			 * Variable to track if we hover over the breadcrumb
			 */
			hovering: false,
			/**
			 * Variable for the open state of the Actions menu
			 */
			open: false,
		}
	},
```

so Vue is fine with mutating it.

Anyway, I will rebase now. Feel free to push any changes which you find necessary.

Reply:
Ah right! Then it won't work.
There is no binding between the open prop of Breadcrumb to the Actions.
You cannot do `<Breadcrumb :open="true">` anymore.
My idea was to completely get rid of the intermediate open within the Breadcrumb. Just forward the prop to the Actions directly :)

Reply:
> You cannot do `<Breadcrumb :open="true">` anymore.

That's true. Wasn't sure if we want this.

> My idea was to completely get rid of the intermediate open within the Breadcrumb. Just forward the prop to the Actions directly :)

Do you mind implementing this? I think you have a better idea how to achieve it already.